vim/core/oni-core-interop/plugin/init.vim

```diff
@@ -110,10 +110,7 @@ let context.windowTopLine = line("w0")
 let context.windowBottomLine = line("w$")
 let context.byte = line2byte(line(".")) + col(".")
 let context.filetype = eval("&filetype")
-
-if exists("b:last_change_tick")
```

Comment:
Can we keep `version` here? The `version` seems like the right thing to send to the language servers, but `&modified` is definitely want we want for actually knowing if the buffer is dirty.